pages/company/visualizations/gh_company_affiliation.py

```diff
@@ -0,0 +1,280 @@
+from dash import html, dcc, callback
+import dash
+import dash_bootstrap_components as dbc
+from dash.dependencies import Input, Output, State
+import plotly.graph_objects as go
+import pandas as pd
+import logging
+from dateutil.relativedelta import *  # type: ignore
+import plotly.express as px
+from pages.utils.graph_utils import color_seq
+from queries.company_query import company_query as cq
+import io
+from cache_manager.cache_manager import CacheManager as cm
+from pages.utils.job_utils import nodata_graph
+import time
+import datetime as dt
+from fuzzywuzzy import fuzz
+
+PAGE = "company"
+VIZ_ID = "gh-company-affiliation"
+
+paramter_1 = "company-contributions-required"
+paramter_2 = "null-check"
+
+
+gc_gh_company_affiliation = dbc.Card(
+    [
+        dbc.CardBody(
+            [
+                html.H3(
+                    "Company Affiliation by Github Account Info",
+                    className="card-title",
+                    style={"textAlign": "center"},
+                ),
+                dbc.Popover(
+                    [
+                        dbc.PopoverHeader("Graph Info:"),
+                        dbc.PopoverBody(
+                            "This graph looks at github contributors profiles and takes in their listed company"
```

Comment:
"This graph uses company affiliations self-reported by users in their Github profile."